apps/site/src/theme/components/data-display/mui-typography-theme-options.ts

```diff
@@ -10,6 +10,7 @@ export const MuiTypographyThemeOptions: Components<Theme>["MuiTypography"] = {
       bpHeading3: "h3",
       bpHeading4: "h4",
       bpHeading5: "h5",
+      bpHeading5: "h6",
```

Comment:
```suggestion
      bpHeading6: "h6",
```

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/1183/commits/47536be3c743ca001960adf17321d1ddebdaed16